project/src/main/java/com/lambda/javaorders/models/Customers.java

```diff
@@ -0,0 +1,171 @@
+package com.lambda.javaorders.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "customers")
+public class Customers
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long customerid;
+
+    @Column(nullable = false)
+    private String custname;
+    private String custcity;
+    private String workingarea;
+    private String custcountry;
+    private String grade;
+    private double openingamt;
+    private double receiveamt;
+    private double paymentamt;
+    private double outstandingamt;
+    private String phone;
+
+    //one to many relationships
+    @ManyToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "agentid", nullable = false)
+    @JsonIgnoreProperties({"agents", "hibernateLAZYInitializer"})
+    private Agents agents = new Agents();
```

Comment:
You don't have to initialize the agent here - it may be best to leave it null until there is an agent associated with this customer